Portfolio/index.html

```diff
@@ -30,6 +46,7 @@
```

Comment:
The closing curly brace for the '.abtme' class is missing. Add a closing curly brace before '.abtme img'.